apps/src/music/views/PatternAiPanel.tsx

```diff
@@ -49,15 +49,22 @@ type GenerateStateType = 'none' | 'generating' | 'error';
 
 const defaultAiTemperature = 8;
 
+// When generating, generatingScanStep goes from 1 to this value.  The first
+// PATTERN_AI_NUM_SEED_EVENTS of these values lights up a seed column, and the
+// remainder give a little delay before the generating help text is shown.
+const numberScanSteps = 18;
```

Comment:
This is clever! Wonder if it would be even clearer to do something like `numerScanSteps = PATTERN_AI_NUM_SEED_EVENTS + 10`?

Reply:
Nice, will do.